modules/@apostrophecms/i18n/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  options: {
+    locales: {
+      en: {
+        label: 'English'
+      },
+      fr: {
+        label: 'French',
+        prefix: '/fr'
+      }
+      // 'en-CA': {
```

Comment:
Not sure about what do do of this for now? I guess we can delete it.

Reply:
It's OK to remove the other locales.